static/i18n/en/app.json

```diff
@@ -73,7 +73,9 @@
     }
   },
   "byteSize": {
-    "kbUnit": "{{size}}Kb"
+    "bytes": "{{size}}bytes",
+    "kbUnit": "{{size}}Kb",
+    "mbUnit": "{{size}}Mb"
```

Comment:
Aren't we suppose to have a space between size and unit?

Reply:
Yes please :)